core/domain/html_cleaner.py

```diff
@@ -91,6 +91,11 @@ def clean(user_submitted_html):
     oppia_custom_tags = (
         rte_component_registry.Registry.get_tag_list_with_attrs())
 
+    # Add 'id-with-value' in oppia custom tags.
+    (
+        oppia_custom_tags[u'oppia-noninteractive-image'].
```

Comment:
Why are we doing this here? Can't we make this happen in `get_tag_list_with_attrs`?

Reply:
yes we can, done.